### PR TITLE
remove tappedInfoBubble helper

### PR DESCRIPTION
**Pull request by pvinis**
The type of this PR is: **Enhancement**

### Description

Removing the use of tappedInfoBubble helper.

Sibling PR https://github.com/artsy/cohesion/pull/235

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

- removing the use of a cohesion helper - pavlos

</details>

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
### This PR contains the following changes:

- Dev changes (removing the use of a cohesion helper - pavlos)
<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 824d71febb30b4ff4c42b8b6bfbc7cd863ed172c
</p>

File: docs/analytics_and_tracking.md [hunk 1]
```diff
@@ -115,3 +109,32 @@ export class MyClassComp extends React.Component<Props> {
...
+        "subject": "demandIndex",
+      },
+    ]
+  `)
+})
+```
+
+You can start with an empty snapshot like `expect(trackEvent.mock.calls[0]).toMatchInlineSnapshot()` and when you run the tests, jest will fill it in. Then check if it is correct, and you are ready to commit.
+
+If at some point the track properties change, then the snapshot will need to be updated. If there is a breakage and for some reason a property is not sent, then this snapshot will alert us correctly.
```
Comment (mdole) [inline docs/analytics_and_tracking.md:140]: niiiiice. this might be the first time that snapshots have ever actually sounded like a good idea to me 😁

**Review by mdole — APPROVED** [review]
👏 awesome work @pvinis. Looking forward to sharing this at platform practice tomorrow!

File: package.json [hunk 2]
```diff
@@ -91,7 +91,7 @@
     "cheerio": "0.22.0"
   },
   "dependencies": {
-    "@artsy/cohesion": "2.22.0",
+    "@artsy/cohesion": "3.2.0-canary.235.3368.0",
```
Comment (pvinis) [inline package.json:94]:
```suggestion
    "@artsy/cohesion": "2.22.0",
```

File: yarn.lock [hunk 3]
```diff
@@ -106,10 +106,10 @@
...
     "@algolia/requester-common" "4.10.4"
 
-"@artsy/cohesion@2.22.0":
-  version "2.22.0"
-  resolved "https://registry.yarnpkg.com/@artsy/cohesion/-/cohesion-2.22.0.tgz#c7c13371bd9cb1babb44ddfa0f57eba0c860cfbc"
-  integrity sha512-VCA2y6m/lAw+TV0kUtRhwOmu54fiRjD7+IhbPF15ZvJsQX1Tn2mcVUqwd957o2CyAZ7ZFkLGhkg/CCKM9Fx6Uw==
+"@artsy/cohesion@3.2.0-canary.235.3368.0":
+  version "3.2.0-canary.235.3368.0"
+  resolved "https://registry.yarnpkg.com/@artsy/cohesion/-/cohesion-3.2.0-canary.235.3368.0.tgz#4f103733658f4040af985c80e7fc0e16b5eebf1e"
+  integrity sha512-exBjzv2dyVjaMnic8Z6LYwXM+xPYovGtNtvpMlT5Y4gtufU0t5NJd6L/iYIAtlt+8KkGfCFgkh0kakaTfBR5Cg==
```
Comment (pvinis) [inline yarn.lock:112]:
```suggestion
"@artsy/cohesion@2.22.0":
  version "2.22.0"
  resolved "https://registry.yarnpkg.com/@artsy/cohesion/-/cohesion-2.22.0.tgz#c7c13371bd9cb1babb44ddfa0f57eba0c860cfbc"
  integrity sha512-VCA2y6m/lAw+TV0kUtRhwOmu54fiRjD7+IhbPF15ZvJsQX1Tn2mcVUqwd957o2CyAZ7ZFkLGhkg/CCKM9Fx6Uw==
```

**Pull request merged**